### PR TITLE
Doc for SetKW

### LINKED ISSUE #324: `SetKW` docs

### PR DESCRIPTION
**Pull request by ituaijagbone**
this commit fixes #324

### CODE REVIEW

File: arrow-docs/docs/docs/datatypes/setkw/README.md [hunk 1]
```diff
@@ -6,4 +6,52 @@ permalink: /docs/datatypes/setkw/
...
+val oldNumbers = setOf( -11, 1, 3, 5, 7, 9).k()
+val evenNumbers = setOf(-2, 4, 6, 8, 10).k()
+val integers = setOf(-5, -4, -3, -2, -1, 0, 1, 2, 3, 4, 5).k()
+```
+SetKW derives the following typeclasses:
+
+[Semigroup](/docs/typeclasses/semigroupk/):
+```kotlin:ank
+val uniqueNaturalNumbers_1 = oldNumbers.combineK(evenNumbers.combineK(integers))
+val uniqueNaturalNumbers_2 = oldNumbers.combineK(evenNumbers).combineK(integers)
```
Comment (pakoito) [inline arrow-docs/docs/docs/datatypes/setkw/README.md]: You can split these each in one snippet that'll print its result.

File: arrow-docs/docs/docs/datatypes/setkw/README.md [hunk 2]
```diff
@@ -6,4 +6,52 @@ permalink: /docs/datatypes/setkw/
...
+```
+[Foldable](/docs/typeclasses/foldable/):
+```kotlin:ank
+val sum_1 = uniqueNaturalNumbers_1.foldLeft(0){sum, number -> sum + (number * number)}
+val sum_2 = uniqueNaturalNumbers_2.foldLeft(0){sum, number -> sum + (number * number)}
+print(sum_1 == sum_2)
+// true
+```
+[Monoid](/docs/typeclasses/monoidk/):
```
Comment (pakoito) [inline arrow-docs/docs/docs/datatypes/setkw/README.md]: `Monoid` !== `MonoidK`!

File: arrow-docs/docs/docs/datatypes/setkw/README.md [hunk 3]
```diff
@@ -6,4 +6,52 @@ permalink: /docs/datatypes/setkw/
...
+```kotlin:ank
+val sum_3 = SetKW.monoidK().combineK(uniqueNaturalNumbers_1, SetKW.empty()).foldLeft(0){sum, number -> sum + (number * number)}
+val sum_4 = SetKW.monoidK().combineK(SetKW.empty(), uniqueNaturalNumbers_1).foldLeft(0){sum, number -> sum + (number * number)}
+print(sum_3 == sum_4)
+// true
```
Comment (pakoito) [inline arrow-docs/docs/docs/datatypes/setkw/README.md]: SemigroupK and MonoidK are similar to Semigroup and Monoid. You can put long examples of the non-K versions and mention the K versions at the bottom.

File: arrow-docs/docs/docs/datatypes/setkw/README.md [hunk 4]
```diff
@@ -6,4 +6,52 @@ permalink: /docs/datatypes/setkw/
...
-TODO
\ No newline at end of file
+SetKW(Kinded Wrapper) is a higher kinded wrapper around the the Set collection interface. 
+
+It can be created from the Kotlin Set type with a convient `k()` function.
+
+```kotlin:ank
+val uniqueNumbers = setOf(1, 2, 5, 3, 2).k()
+print(uniqueNumbers)
+// SetKW(set=[1, 2, 5, 3])
```
[the remark this answers is not in the document]
Comment (ituaijagbone) [reply arrow-docs/docs/docs/datatypes/setkw/README.md]: Thanks. Fixed

**Comment by raulraja** [comment]
thanks for your contribution @ituaijagbone !

**Pull request merged**